### PR TITLE
feat: woocommerce integration - added product and client webhooks

### PR DESCRIPTION
**Pull request by gamaraan**
Added to woocommerce integration the possibility to have items and clients webhooks, so new item/item updates and new client/client updates events can be synced to ERPNext not only on new orders, but as they occur in woocommerce site.
Webhooks are set up in a similar way with the main order created webhook:

-  for new item/item updated events, pointing to the endpoint your.erp.site/api/method/erpnext.erpnext_integrations.connectors.woocommerce_connection.item
- new client/client updated, pointing to the endpoint your.erp.site/api/method/erpnext.erpnext_integrations.connectors.woocommerce_connection.client

using same secret as set up on new order webhook.

### CODE REVIEW

File: erpnext/erpnext_integrations/connectors/woocommerce_connection.py [hunk 1]
```diff
@@ -20,45 +24,44 @@ def verify_request():
 		not sig == bytes(frappe.get_request_header("X-Wc-Webhook-Signature").encode()):
 			frappe.throw(_("Unverified Webhook Data"))
 	frappe.set_user(woocommerce_settings.creation_user)
+	print ("Request verified successfully")
```
Comment (rmehta) [inline erpnext/erpnext_integrations/connectors/woocommerce_connection.py]: print

**Review by rmehta — CHANGES_REQUESTED** [review]
Please also update documentation for setting up webhooks

:+1: for first PR to ERPNext

File: erpnext/erpnext_integrations/connectors/woocommerce_connection.py (hunk 1 above)
Comment (gamaraan) [reply erpnext/erpnext_integrations/connectors/woocommerce_connection.py]: Should I edit the page here:
https://erpnext.com/docs/user/manual/en/erpnext_integration/woocommerce_integration
?

**Review by gamaraan — COMMENTED** [review]
print removed

**Pull request closed without merging**